test/recipes/30-test_evp_data/evpmac_common.txt

```diff
@@ -67,6 +67,29 @@ BlockSize = 64
 
 Title = SHA2
 
+Availablein = default
+MAC = HMAC
```

Comment:
Please include a comment about where the tests have come from
Do the same with test file below.

Reply:
I computed these with a python script, if you're hoping for a more official source AFAIK there isn't one

Likewise the tests in `evpmd_sha.txt` are just the SHA256 tests with the hash truncated